### PR TITLE
Update gb-chc.json

### PR DESCRIPTION
**Pull request by siwhitehouse**
Updated with the new URLs, added missing data and included Welsh language

### CODE REVIEW

File: lists/gb/gb-chc.json [hunk 1]
```diff
@@ -24,32 +24,44 @@
   "listType": "primary",
   "access": {
     "availableOnline": true,
-    "onlineAccessDetails": "",
-    "publicDatabase": "https://www.gov.uk/government/organisations/charity-commission",
-    "guidanceOnLocatingIds": "",
+    "onlineAccessDetails": "Information about the data and information available through the search is available on the page listed as the list URL. Click through to \"Find a charity\" for a simple text search of a charity's name. Use the advanced search to filter the search across multiple different aspects of the dataset.",
+    "publicDatabase": "https://register-of-charities.charitycommission.gov.uk/",
+    "guidanceOnLocatingIds": "Identifiers are listed as \"Charity numbers\" and are shown in the returned list view to a search.",
```
Comment (kd-ods) [inline lists/gb/gb-chc.json:29]: Suggested edit: replace 'returned list view to a search' with 'list returned by a search'

File: lists/gb/gb-chc.json [hunk 2]
```diff
@@ -24,32 +24,44 @@
   "listType": "primary",
   "access": {
     "availableOnline": true,
-    "onlineAccessDetails": "",
-    "publicDatabase": "https://www.gov.uk/government/organisations/charity-commission",
-    "guidanceOnLocatingIds": "",
+    "onlineAccessDetails": "Information about the data and information available through the search is available on the page listed as the list URL. Click through to \"Find a charity\" for a simple text search of a charity's name. Use the advanced search to filter the search across multiple different aspects of the dataset.",
```
Comment (kd-ods) [inline lists/gb/gb-chc.json:27]: Suggestion: replace '...information available through the search is available on the page listed as the list URL' with 

'...information available through the search is available at https://register-of-charities.charitycommission.gov.uk'.

**Review by kd-ods — COMMENTED** [review]
There are a couple of editing suggestions.

File: lists/gb/gb-chc.json (hunk 2 above)
Comment (siwhitehouse) [reply lists/gb/gb-chc.json]: Thanks @kd-ods those suggestions make the text a lot clearer. I've made the changes as recommended.

File: lists/gb/gb-chc.json [hunk 3]
```diff
@@ -24,32 +24,44 @@
   "listType": "primary",
   "access": {
     "availableOnline": true,
-    "onlineAccessDetails": "",
-    "publicDatabase": "https://www.gov.uk/government/organisations/charity-commission",
-    "guidanceOnLocatingIds": "",
+    "onlineAccessDetails": "Information about the data and information available through the search is available available at https://register-of-charities.charitycommission.gov.uk'. Click through to \"Find a charity\" for a simple text search of a charity's name. Use the advanced search to filter the search across multiple different aspects of the dataset.",
```
Comment (kd-ods) [inline lists/gb/gb-chc.json:27]: Typo: "available available"

**Review by kd-ods — COMMENTED** [review]
Sorry - just spotted a typo!

File: lists/gb/gb-chc.json (hunk 3 above)
Comment (siwhitehouse) [reply lists/gb/gb-chc.json:27]: That's because it's *very* available ;0)

**Review by kd-ods — APPROVED** [review]
Looks good.

**Pull request merged**